src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp

```diff
@@ -691,8 +691,7 @@ void SinglePhaseBase::implicitStepSetup( real64 const & GEOS_UNUSED_PARAM( time_
     {
       saveConvergedState( subRegion );
 
-      arrayView1d< real64 > const & dVol = subRegion.template getField< fields::flow::deltaVolume >();
-      dVol.zero();
+      applyDeltaVolume( subRegion );
```

Comment:
the main fix is here

Reply:
the fix is not needed when defaultAperture is consistent with hydraulic aperture table
still it provides some safeguard when it is not consitent and make hydraulic aperture table override what what specified in defaultAperture
@CusiniM, @Guotong-Ren, @rrsettgast please let me know if it makes sense to merge it?